digitalocean/resource_digitialocean_database_connection_pool.go

```diff
@@ -0,0 +1,192 @@
+package digitalocean
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/digitalocean/godo"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceDigitalOceanDatabaseConnectionPool() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDigitalOceanDatabaseConnectionPoolCreate,
+		Read:   resourceDigitalOceanDatabaseConnectionPoolRead,
+		// Update: resourceDigitalOceanDatabaseConnectionPoolUpdate,
+		Delete: resourceDigitalOceanDatabaseConnectionPoolDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"db_id": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"name": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"user": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"mode": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"size": {
+				Type:         schema.TypeInt,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"database": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.NoZeroValues,
+			},
+
+			"pool_host": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"pool_port": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+
+			"pool_uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"pool_database": {
```

Comment:
`pool_database` and `pool_user` should always be the same as the `database` and `user`. I don't think there is a need to export them as separate attributes.

Reply:
Absolutely! I added the prefix to avoid a conflict between the computed `database` and the argument `database`. However, as you pointed out, they're always the same. It was late in the evening.

I'll patch it up :-)